module/solutions/helper.js

```diff
@@ -2404,6 +2413,70 @@ module.exports = class SolutionsHelper {
     });
   }
 
+   /**
+   * List Solutions using organization id.
+   * @method
+   * @name queryForOrganizationSolutions
+   * @query {String} type - Assets type (program/solutions).
+   * @returns {Object} - Details of the solution under the organization.
+   */
+
+   static queryForOrganizationSolutions(bodyData,queryData){
+
+    return new Promise(async (resolve, reject) => {
+      try {
+        let pageNo = "";
+        let pageSize = "";
+        let searchText = "";
+        let type = "";
+        let subType = "";
+        let matchQuery = {};
+        let filterEmptyStringsFromArray;
+
+        //if there is an empty string then remove from userids
+        if (!bodyData.filters.userId) {
+          filterEmptyStringsFromArray = [];
+        } else {
+          filterEmptyStringsFromArray = bodyData.filters.userId.filter(
+            (str) => str !== ""
+          );
+        }
+
+        if (filterEmptyStringsFromArray.length > 0) {
+          matchQuery = {
+            $and: [
+              { createdFor: { $in: [bodyData.filters.orgId] } },
+              { author: { $in: bodyData.filters.userId } },
+              { isAPrivateProgram: false },
+            ],
+          };
+        } else {
+          matchQuery = { createdFor: { $in: [bodyData.filters.orgId] } };
+        }
+        
+        let solutionDocuments = await this.list(
+          type,
```

Comment:
you can pass "" here only and add comment why you are passing "" empty string @praveenKDass

Reply:
resolved